packages/dscc-gen/src/viz/index.ts

```diff
@@ -38,15 +37,12 @@ export const createFromTemplate = async (
     {match: /{{PROD_BUCKET}}/g, replace: prodBucket!},
   ];
   await files.fixTemplates(projectPath, templates);
-
-  await util.spinnify('Installing project dependencies...', async () => {
-    if (config.yarn) {
-      await execa('yarn', [], {cwd: projectPath});
-    } else {
-      await execa('npm', ['install'], {cwd: projectPath});
-    }
-  });
-
+  console.log('Installing dependencies...');
```

Comment:
We don't want this to be a console.log, part of spinnify is that it removes the text after the step is done. This should probably still be done in spinnify.

Reply:
Spinnify expects async behavior.. which was the problem with viz in the first place. Trying, again, though.